test/test_add_spec.lua

```diff
@@ -14,7 +14,7 @@ local function date_test(test, func)
     local format = icu_date.formats.iso8601()
 
     date:set_millis(1507836727123)
-    test:ok("2017-10-12T19:32:07.123Z" == date:format(format))
+    test:is("2017-10-12T19:32:07.123Z", date:format(format))
```

Comment:
The `test:is()` function [accepts](https://github.com/tarantool/tarantool/blob/b0431cf8f47e9d081f6a402bc18edb1d6ad49847/src/lua/tap.lua#L142) the actual value as the first and the expected value as the second.

Reply:
Fixed